src/defaults.js

```diff
@@ -0,0 +1,102 @@
+"use strict";
+const { defaultOptions } = require("./utils");
+const { sign } = require("tweetnacl");
+const { toByteArray } = require("base64-js");
+
+BigInt.prototype.toJSON = function () {
+  return this.toString();
+};
+
+const defaultBaseOptions = {
+  APIKey: "",
+  skynetApiKey: "",
+  customUserAgent: "",
+  customCookie: "",
+  onDownloadProgress: undefined,
+  onUploadProgress: undefined,
+  loginFn: undefined,
+};
+
+const defaultGetEntryOptions = {
+  ...defaultBaseOptions,
+  endpointGetEntry: "/skynet/registry",
+  hashedDataKeyHex: false,
```

Comment:
This field should be removed since it appears unused. Same below in `defaultSetEntryOptions`.

Reply:
In the new version it is used, but with the original name of "skydb-js".

DEFAULT_GET_ENTRY_OPTIONS

Or did they mean something else?